### PR TITLE
[TITS-5138] Inject GTM into the page

### PR DESCRIPTION
**Pull request by kudlajz**

### CODE REVIEW

File: modules/analytics/components/Analytics.tsx [hunk 1]
```diff
@@ -85,6 +86,27 @@ function Analytics() {
         }
     });
 
+    if (newsroom?.ga_tracking_id) {
+        return (
+            <Head>
+                <script
+                    async
+                    src={`https://www.googletagmanager.com/gtag/js?id=${newsroom.ga_tracking_id}`}
+                />
```
Comment (oleg-semyonov) [inline modules/analytics/components/Analytics.tsx:95]: There is a [Script](https://nextjs.org/docs/basic-features/script) component in Next that can be used here instead :)

Comment (oleg-semyonov) [inline modules/analytics/components/Analytics.tsx:95]: There is a [Script](https://nextjs.org/docs/basic-features/script) component in Next that can be used here instead :)

Comment (oleg-semyonov) [reply modules/analytics/components/Analytics.tsx:95]: Also, Gijs mentioned that we support both the new GTag identifers and the older `GA-` IDs. I believe those require a different script, please check with the GA docs

**Review by oleg-semyonov — COMMENTED** [review]
Great stuff! I have a couple suggestions :)

File: modules/analytics/components/Analytics.tsx (hunk 1 above)
Comment (kudlajz) [reply modules/analytics/components/Analytics.tsx]: I've been into Google Analytics dashboard and the tracking code is the same, only the ID changes as far as I can see. I've tested it with both and I've seen some traffic in Google Analytics dashboard but I will check it after deployment as well.

I think you've meant the older `UA-` IDs. The newer ones are `G-`.

Comment (oleg-semyonov) [reply modules/analytics/components/Analytics.tsx]: Right, my bad :)
Good that it works!

**Review by oleg-semyonov — APPROVED** [review]

**Pull request merged**